earthly/rust/scripts/std_build.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# cspell: words lcov testdocs nextest testunit depgraph
+
+import python.cli as cli
+import argparse
+import rich
+
+# This script is run inside the `build` stage.
+# This is set up so that ALL build steps are run and it will fail if any fail.
+# This improves visibility into all issues that need to be corrected for `build`
+# to pass without needing to iterate excessively.
+
+def main():
+    # Force color output in CI
+    rich.reconfigure(color_system="256")
+
+    parser = argparse.ArgumentParser(
+        description="Rust build processing."
+    )
+    parser.add_argument("--target", default="", help="Pass rust --target flag (cargo --target flag).")
+    parser.add_argument("--cov_report", default="", help="The output coverage report file path.")
+    parser.add_argument("--with_bench", default="false", help="Running benchmarks")
+    args = parser.parse_args()
+
+    results = cli.Results("Rust build")
+
+    target_flag = ""
+    if args.target != "":
+        target_flag += f" --target={args.target} "
+
+    # Build the code.
+    results.add(cli.run("cargo build " \
+                        f"{target_flag} " \
+                        "--release ",
```

Comment:
```suggestion
                        + "--release ",
```

Reply:
Don't suppress line breaks, join strings using `+` across multiple lines.
This applies to all instances of this in the PR